03-client-api/references/options.yml

```diff
@@ -0,0 +1,94 @@
+
+title: Options
+description:
+  common: >
+    The query options are set to specific defaults in the server when not explicitly declared by the user.
+  java: >
+    The transaction query options object `Transaction.Options` can be used to override the default server behaviour
+    query processing.
+
+    The server defaults are:
+    `infer = true`
+    `explain = false`
+    `batchSize = 50`
+
+  python: >
+    The transaction query optional parameters can be used to override the default server behaviour for query processing.
+    The server defaults are:
+
+    `infer = true`
+    `explain = false`
+    `batch_size = 50`
+
+    `batch_size` tells the server to pre-compute and stream this number of answers at a time. These are buffered in the
+    client until read. A larger batch size causes the server to compute more answers at a time, blocking the transaction
+    until the batch is computed.
+
+  javascript: >
+    The transaction query options Object can be used to override the default server behaviour for query processing.
+    Provide an object mapping the allowed keys: `infer`, `explain`, and `batchSize`.
+
+    The server defaults are:
+    `infer = true`
+    `explain = false`
+    `batchSize = 50`
+
+    `batchSize` tells the server to pre-compute and stream this number of answers at a time. These are buffered in the
+    client until read. A larger batch size causes the server to compute more answers at a time, blocking the transaction
+    until the batch is computed.
```

Comment:
I'm pretty sure `common` is not prepended to language-specific descriptions. so with the way you have defined the descriptions, I believe the text `The query options are set to specific defaults in the server when not explicitly declared by the user.` would not be displayed, and instead is overridden by each language's description.

Reply:
Ok, I'll just copy paste it into each one and get rid of common